mcs/class/Mono.Posix/Mono.Unix.Native/Syscall.cs

```diff
@@ -727,6 +727,194 @@ public enum MremapFlags : ulong {
 		MREMAP_MAYMOVE = 0x1,
 	}
 
+	[Map]
+	[CLSCompliant (false)]
+	public enum SockType : int {
```

Comment:
This should be named `SocketType`, as per [naming conventions](https://github.com/mono/mono/blob/3619fe9c4c88b87f95dfe7fe967d15df281fdb21/mcs/class/Mono.Posix/Mono.Unix.Native/Syscall.cs#L1582-1588).